packages/dashboards/src/lib/components/widget/widget-header/widget-header.component.ts

```diff
@@ -55,6 +56,8 @@ export class WidgetHeaderComponent implements OnInit, OnDestroy, AfterViewInit {
     }
 
     @ViewChild("widgetHeaderCustomElement") public widgetHeaderCustomElement: ElementRef;
+    @ViewChild("link") public linkElement: any;
```

Comment:
can it be this instead?

```suggestion
    @ViewChild("link") public linkElement: ElementRef;
```